From babfa64585bef6c53c1d399a1c244aea89437c9e Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Sat, 9 Sep 2023 23:58:50 +0200
Subject: [PATCH] HACK: keep _ISOC2X_SOURCE undefined to build localedef with
 old glibc

Fixes:
host-localedef-2.38-13-g92201f16cbcfd9eafe314ef6654be2ea7ba25675/build/locale/ld-ctype.o: in function `ctype_read':
ld-ctype.c:(.text+0x6cef): undefined reference to `__isoc23_strtoul'

Upstream: Not applicable

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 include/features.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/features.h b/include/features.h
index 7c51b4a2e4..f385370d18 100644
--- a/include/features.h
+++ b/include/features.h
@@ -207,7 +207,6 @@
 # undef  _ISOC11_SOURCE
 # define _ISOC11_SOURCE	1
 # undef  _ISOC2X_SOURCE
-# define _ISOC2X_SOURCE	1
 # undef  _POSIX_SOURCE
 # define _POSIX_SOURCE	1
 # undef  _POSIX_C_SOURCE
-- 
2.41.0

